docs/guides/ferretdb/concepts/ferretdb.md

```diff
@@ -235,9 +229,29 @@ Following table show what KubeDB does when you delete FerretDB CR for different
 
 If you don't specify `spec.deletionPolicy` KubeDB uses `Delete` deletion policy by default.
 
-### spec.podTemplate
+### spec.server
+
+After FerretDB version 2.0.0, FerretDB uses PostgreSQL + DocumentDB extension as the database storage and currently supports replication using the Write-Ahead Logging (WAL) streaming method.
```

Comment:
Refer the db.spec.storage field in this page. That will be used for created-pg-storage